tests/client/api_events/test_milestones_confirmed_event.c

```diff
@@ -97,7 +97,8 @@ int main() {
   UNITY_BEGIN();
 
   RUN_TEST(test_milestones_confirmed_parser);
-  RUN_TEST(test_milestones_confirmed_events);
+  // For local tests only
+  // RUN_TEST(test_milestones_confirmed_events);
```

Comment:
Can add a switch to enable all online tests, like `test_find_message.c` test case?
or use `TEST_TANGLE_ENABLE` in `api_v1/test_config.h`?

Reply:
Moved tests/client/api_v1/test_config.h to tests/client/test_config.h and added event configs also to test_config.h, so both api_events and api_v1 uses test_config.h